hpat/tests/test_series.py

```diff
@@ -2031,27 +2030,264 @@ def test_series_str2str(self):
+        '''Verify Series.append() concatenates Series with list of other Series ignoring index'''
+        def test_impl(S1, S2, S3):
+            return S1.append([S2, S3], ignore_index=True)
+        hpat_func = hpat.jit(test_impl)
+
+        dtype_to_data = {'float': [[-2., 3., 9.1], [-2., 5.0], [1.0]]}
+        if not hpat.config.config_pipeline_hpat_default:
+            dtype_to_data['string'] = [['a', None, ''], ['d', None], ['']]
+
+        for dtype, data_list in dtype_to_data.items():
+            with self.subTest(series_dtype=dtype, concatenated_data=data_list):
+                S1, S2, S3 = [pd.Series(data) for data in data_list]
+                pd.testing.assert_series_equal(hpat_func(S1, S2, S3), test_impl(S1, S2, S3))
+
+    @unittest.skip('BUG: Pandas 0.25.1 Series.append() doesn\'t support tuple as appending values')
+    def test_series_append_tuple_ignore_index(self):
+        '''Verify Series.append() concatenates Series with tuple of other Series ignoring index'''
+        def test_impl(S1, S2, S3):
+            return S1.append((S2, S3, ), ignore_index=True)
+        hpat_func = hpat.jit(test_impl)
+
+        dtype_to_data = {'float': [[-2., 3., 9.1], [-2., 5.0], [1.0]]}
+        if not hpat.config.config_pipeline_hpat_default:
+            dtype_to_data['string'] = [['a', 'b', 'q'], ['d', 'e'], ['s']]
+
+        for dtype, data_list in dtype_to_data.items():
+            with self.subTest(series_dtype=dtype, concatenated_data=data_list):
+                S1, S2, S3 = [pd.Series(data) for data in data_list]
+                pd.testing.assert_series_equal(hpat_func(S1, S2, S3), test_impl(S1, S2, S3))
+
+    @unittest.skipIf(hpat.config.config_pipeline_hpat_default,
+                     "BUG: old-style append implementation doesn't handle series index")
+    def test_series_append_single_index_default(self):
+        '''Verify Series.append() concatenates Series with other single Series supporting default index'''
+        def test_impl(S, other):
+            return S.append(other)
+        hpat_func = hpat.jit(test_impl)
+
+        dtype_to_data = {'float': [[-2., 3., 9.1], [-2., 5.0]]}
+        if not hpat.config.config_pipeline_hpat_default:
+            dtype_to_data['string'] = [['a', 'b', 'q'], ['d', 'e']]
+
+        for dtype, data_list in dtype_to_data.items():
+            with self.subTest(series_dtype=dtype, concatenated_data=data_list):
+                S1, S2 = [pd.Series(data) for data in data_list]
+                pd.testing.assert_series_equal(hpat_func(S1, S2), test_impl(S1, S2))
+
+    @unittest.skipIf(hpat.config.config_pipeline_hpat_default,
+                     "BUG: old-style append implementation doesn't handle series index")
+    def test_series_append_list_index_default(self):
+        '''Verify Series.append() concatenates Series with list of other Series ignoring index'''
+        def test_impl(S1, S2, S3):
+            return S1.append([S2, S3])
```

Comment:
The test description says 'ignoring index'. So, either you should add `ignore_index=True` or change test description

Reply:
Yes, thank you, I'll correct all the tests docstring in the next commit